doc/filter_tutorial.md

```diff
@@ -0,0 +1,376 @@
+# Alerts tutorial
+
+This document will help you navigate through the simulated Rubin alerts ingested in the MongoDB Compass, so that you can try different filtering schemes using the MongoDB syntax, which is compatible with Fritz.
+First a summary of what you’ll do:
+- Install MongoDB Compass
+- Connect to cluster with Rubin alerts
+- Create your filter
+- Save your candidates
+- Explore your results
+
+### 0. Note
+I based most of this work on the tutorial you can find here, so, if you have any issues, I would start looking them up there.
+### 1. To install MongoDB Compass
+ Go to https://www.mongodb.com/products/compass and click in ‘Download now’
+### 2. Connecting to cluster
+Once Mongo DB Compass is installed, launch it, and connect to the cluster by
+
+   - Opening Mongo DB Compass
+   - Copy the following into the box under New Connection, and click Connect.  <br/>
+mongodb+srv://alert_digger:WhereIsTheKN@cluster0.s0ada.mongodb.net/test  <br/>
+
+
+     ![](https://github.com/tahumada/fritz/blob/main/doc/data/filter_examples/tutorial1.png)
+
+
+
+ - To see the alerts (around 4000 of them) navigate in the left menu to the folder ZTF > lsst_800
+
+   ![This is an image](https://github.com/tahumada/fritz/blob/main/doc/data/filter_examples/tutorial2.png)
```

Comment:
There must be a way to point the image links to local files / relative paths so the images would live on Fritz and not on your fork.

Reply:
I pushed also the images, and now I reference them as if they lived on Fritz (not sure if I did it correctly)